parse_lcov_total.py

```
@@ -0,0 +1,33 @@
+import sys
+import os
+
+def parse_lcov(filepath):
+    lf = 0
+    lh = 0
+    try:
+        with open(filepath, 'r') as f:
```

Ruff (0.15.4)

[warning] Unnecessary mode argument

Remove mode argument

(UP015)